src/components/list.js

```diff
@@ -1,41 +1,61 @@
-"use client";
+'use client'
 
 import React from 'react';
 import Link from "next/link";
 import usePokemon from '@/hooks/usePokemon';
 import Image from "next/image";
 
 export default function list() {
-
-  const { pokemonData } = usePokemon();
+  const { filteredPokemon, searchQuery, handleSearchChange } = usePokemon();
 
   return (
     <>
+      <div className="mx-20 py-auto ">
+        <div className="relative">
+          <div className="absolute inset-y-0 flex items-center start-0 ps-3">
+            <img src="./search.svg" alt="search-icon" />
+          </div>
+          <div className="flex">
+            <input
+              type="search"
+              id="search"
+              className="block w-full p-2 text-sm text-black rounded-full ps-10 focus:outline-none"
+              placeholder="Search"
+              value={searchQuery}
+              onChange={handleSearchChange}
+              required
+            />
+            <img className="p-2 mx-4 bg-white rounded-full" src="./text_format.svg" alt="" />
```

Comment:
also this one

Reply:
okay, i will change to Image from nextJs